modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -20,111 +20,129 @@ However, the root user on {ubuntu} is disabled by default, so to use bootstrappi
 
 
 
-== Prepare to Register
+== Add Software Channels
 
-Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
+Before you register {ubuntu} clients to your {productname} Server, check that you have the {ubuntu} product enabled, and the required channels are fully synchronized.
 
 
 ifeval::[{suma-content} == true]
-.Procedure: Adding Client Tools Channels
 
-Before you begin, ensure you have the {ubuntu} product enabled, and have synchronized the {ubuntu} channels for {scc}:
+The channels you need for this procedure are:
 
-You can do this from {webui} or using command prompt, at your choice.
+[[ubuntu-channels-wizard]]
+[cols="1,1,1,1,1", options="header"]
+.Ubuntu Channels - WebUI
+|===
+| OS Version
+| Base Channel
+| Security Channel
+| Updates Channel
+| Universe Channel
 
-For {ubuntu} 20.04:
+| {ubuntu} 20.04
+| Ubuntu 20.04
```

Comment:
This is not the Base Channel, it is the Product name you see in the Product page. The base channel is named `ubuntu-2004-amd64-main for amd64 ` and is invisible in the product page. You can see it only "after" you mirrored the product in the channel view.